docs/conf.py

```diff
@@ -41,7 +41,7 @@
 
 # General information about the project.
 project = u'treq'
-copyright = u'2014, David Reid'
+copyright = u'2014–2020 David Reid'
```

Comment:
Do we need an updated copyright holder?

Reply:
Let's update this separately.  Especially because dreid probably doesn't have copyright over any of the bits that are new in 2020? And also there do not appear to be any new bits in 2020 just looking over the commit logs on my local `master` so this is doubly inaccurate.

Reply:
I was planning to update to say "treq Authors" and list the authors in the LICENSE file as Twisted does. I don't want to do that as part of this PR as it will involve e-mailing all of the contributors to check if they are the copyright holders of their contributions.

This PR does include some documentation change that are copyrightable (e.g., the changelog entries I wrote) so I think that bumping the date here is reasonable.

Reply:
> I was planning to update to say "treq Authors" and list the authors in the LICENSE file as Twisted does. I don't want to do that as part of this PR as it will involve e-mailing all of the contributors to check if they are the copyright holders of their contributions.

Clearly the only reasonable choice :-).